AllReadyApp/Web-App/AllReady/Areas/Admin/Views/Invite/Send.cshtml

```diff
@@ -0,0 +1,56 @@
+﻿@model AllReady.Areas.Admin.ViewModels.Invite.InviteViewModel
+
+@{
+    string formAction;
+    if (Model != null && Model.InviteType == AllReady.Areas.Admin.ViewModels.Invite.InviteType.EventManagerInvite)
```

Comment:
when would Model ever be null? You're explicitly creating it in the Get action method. Remove this check